docs/docs/schema.md

```diff
@@ -0,0 +1,367 @@
+---
+
+title: Schema
+description: Define the schema of your database.
+
+---
+
+## Overview
+
+In Drift, your schema is represented as a table. Each column in the table represents a field in your data class.
+
+Take this example of a table that stores superheroes:
+
+| ID  | Name      | Secret Name  | Age    | Height |
+| --- | --------- | ------------ | ------ | ------ |
+| 1   | Superman  | Clark Kent   | 35     | 6'2"   |
+| 2   | Batman    | Bruce Wayne  | 40     | 6'0"   |
+| 3   | Spiderman | Peter Parker | 25     | 5'10"  |
+| 4   | Ironman   | Tony Stark   | `null` | 6'1"   |
+| 5   | Thor      | `null`       | 1500   | `null` |
+
+
+Here we have a table with 5 columns: `ID`, `Name`, `Secret Name`, `Age`, and `Height`. Each row in the table represents a single superhero.
+
+Also, we need to limit how the data is stored, for instance:
+
+- `ID` and `Name` should be unique for each row.
+- `Secret Name`, `Age`, and `Height` are optional fields.
+
+You can easily define a table in Drift using the following syntax:
+
+{{ load_snippet('superhero_schema','lib/snippets/schema.dart.excerpt.json') }}
```

Comment:
Add a little comment at the top that this file is named `schema.dart`. Otherwise users not familiar with build runner might copy contents into a different file and get confused later.